std_msgs/Header header

ufomap_msgs/AABB[] aabbs

ufomap_msgs/Frustum[] frustums

ufomap_msgs/LineSegment[] line_segments

ufomap_msgs/OBB[] obbs

ufomap_msgs/Plane[] planes

ufomap_msgs/Ray[] rays

ufomap_msgs/Sphere[] spheres